Resources/Locale/en-US/store/uplink-catalog.ftl

```diff
@@ -73,6 +73,12 @@ uplink-mosin-ammo-desc = A box of cartridges for the surplus rifle.
 uplink-sniper-ammo-name = ammunition box (.60 antimateriel)
 uplink-sniper-ammo-desc = a box of 10 cartridges for the hristov sniper rifle
 
+uplink-shotgun-magazine-lethal-name = Lethal bulldog magazine
+uplink-shotgun-magazine-lethal-desc = A box magazine of 8 lethal shells for the bulldog auto shotgun
```

Comment:
Change the name to the actual exact name of the item
The description should not call it a box magazine either, and use a punctuation mark at the end of the description

Reply:
Look up the actual item name from Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Magazines/shotgun.yml 
every other uplink entry uses the exact entity names

and you are still  describing it as a "box magazine", which it is not